src/scicloj/clay/v2/main.clj

```diff
@@ -1,37 +1,54 @@
 (ns scicloj.clay.v2.main
   "command line interface"
   (:gen-class)
-  (:require [clojure.edn :as edn]
+  (:require [babashka.fs :as fs]
             [clojure.tools.cli :as cli]
             [scicloj.clay.v2.api :as api]
             [nrepl.cmdline]))
 
 (def cli-options
-  [["-s" "--source-path PATHS"
-    :default-desc (pr-str ["notebooks"])
-    :parse-fn edn/read-string]
-   ["-t" "--base-target-path DIR"
-    :default-desc "docs"]
-   ["-r" "--live-reload"]
+  [["-t" "--base-target-path DIR" :default-desc "docs"]
+   ["-r" "--render"]
    ["-h" "--help"]])
 
+(def default-options
+  {:live-reload true})
+
+(def render-options
+  {:show        false
+   :serve       false
+   :live-reload false})
+
 (defn -main
-  "Invoke with `clojure -M:dev -m scicloj.clay.v2.main --help` to see options"
+  "Invoke with `clojure -M:dev -m scicloj.clay.v2.main --help` to see options
```

Comment:
I'm unsure if `-M:dev` is necessary here; it seems to be omitted in some places, which is better from a user's perspective.

Reply:
Thanks. 

Technically, `-M` is not needed if Clay is in the main deps, but it does give a warning this way.
```
WARNING: Implicit use of clojure.main with options is deprecated, use -M
```

Reply:
Anyway, thanks for the reminder. I will improve the docstring.